src/components/Slogan/Slogan.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import './Slogan.scss';
+
+export class Slogan extends React.Component {
```

Comment:
Also, this could be a "STUPID" component. We don't need to hold slogan in state.